src/account/accountActions.js

```diff
@@ -6,6 +6,7 @@ import {
   ACCOUNT_REMOVE,
   LOGIN_SUCCESS,
   LOGOUT,
+  SAVE_GCM_TOKEN
```

Comment:
You might want to call it SAVE_TOKEN_GCM as there will be another SAVE_TOKEN_APNS or similar.
In this naming, follow with the main action (save token) and then the details (GCM) though it is not how you would order it if you speak.